packages/ui-components/src/Input/EmailPassword.module.scss

```diff
@@ -19,6 +17,18 @@
   font-size: 12px;
 }
 
-.new-password-validation-label {
+.new-password-validation-icon {
+  padding-top: crl-gutters(0.5);
+  display: inline-flex;
+  opacity: 0.35,
+}
+
+.new-password-validation-error,
+.new-password-validation-success {
   padding-left: crl-gutters(0.75);
+  color: $color-font-2;
+}
+
+.new-password-validation-error {
+  color: $color-font-4;
```

Comment:
we want to have neutral-7 for success validation messages, and neutral-5 for error validation messages as shown in Figma (https://www.figma.com/file/B5AtEGdRbW3VxiBdg62TB8Xw/CRDB-Design-System?node-id=22%3A4).